atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueService.java

```diff
@@ -69,157 +74,191 @@
         return sizeInBytes;
     }
 
-    public static ProfilingKeyValueService create(KeyValueService delegate) {
-        return new ProfilingKeyValueService(delegate);
+    public static ProfilingKeyValueService create(KeyValueService delegate, long slowLogThresholdMillis) {
```

Comment:
This is a breaking change.  I happen to know this will effect one of our products.  May want to instead turn this into an additive change and deprecate this method.